control_plane/input_params/ethernet_vars.yml

```diff
@@ -13,105 +13,122 @@
 #  limitations under the License.
 
 
-# ---Default configurations written for Dell PowerSwitch S4128T-ON---
+# ---Default configurations written for Dell PowerSwitch S5232F-ON---
 # ---Change the configurations as per the switch model to avoid failures---
+# ---Use this configuration file for all switches other than S3* and S4* switches---
+
 
 # Global configuration for switch
 os10_config:
     - "interface vlan1"
     - "exit"
 
 # Interface configuration for switch
+# By default, all ports are brought up in admin UP state
+# By default, all ports are set up in 10g-4x breakout mode
+# Possible values of breakout/fanout mode: 10g-4x, 25g-4x, 40g-1x, 50g-2x, 100g-1x
 os10_interface:
     ethernet 1/1/1:
-      desc: "Port 1"
       admin: up
+      fanout: 10g-4x
```

Comment:
Would it make sense to create a `fanout` variable so that if a customer needs to change the port speed config they can do it in one location instead of 30?

Reply:
@lwilson Agree, we will use breakout variable which is widely used in IB switch as well.